app/Services/StandService.php

```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
     {
         return Stand::where('id', $standId)->exists();
     }
+
+    /**
+     * Delete a given stand
+     *
+     * @param string $airfield
+     * @param string $identifier
+     * @throws \Exception
+     */
+    public function deleteStand(string $airfield, string $identifier)
+    {
+        $stand = Stand::with('airfield')->whereHas(
+            'airfield',
+            function (Builder $query) use ($airfield) {
+                $query->where('code', $airfield);
+            }
+        )->where('identifier', $identifier)
+            ->first();
+
+        if (!$stand) {
+            return;
+        }
+
+        $stand->delete();
+        DependencyService::touchDependencyByKey(self::STAND_DEPENDENCY_KEY);
+    }
+
+    /**
+     * Change the identifier for a stand
+     *
+     * @param string $airfield
+     * @param string $oldIdentifier
+     * @param string $newIdentifier
+     */
+    public function changeStandIdentifier(string $airfield, string $oldIdentifier, string $newIdentifier)
+    {
+        $stand = Stand::with('airfield')->whereHas(
```

Comment:
is this code identical to find the identifiers? Maybe a candidate for a private function

Reply:
Done!